src/releng_notification_identity/releng_notification_identity/__init__.py

```diff
@@ -0,0 +1,10 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+from .flask import app
```

Comment:
can we remove everything from this file after line 5? i just removed this from all projects in master, it was not really working. the way to run flask is now ``FLASK_APP=<project>.flask:app flask run``. it is also documented.